ema/nextjs/src/components/content-types/activity.js

```diff
@@ -12,7 +12,11 @@ function Activity({ title, description, image, urlTitle }) {
         <article className="p-4 overflow-hidden bg-white rounded shadow-lg">
             <Image
                 className="w-full"
-                src={`${process.env.NEXT_PUBLIC_DOTCMS_HOST}${image}?language_id=${language.id}`}
+                src={
+                    image
+                        ? `${process.env.NEXT_PUBLIC_DOTCMS_HOST}${image}?language_id=${language.id}`
+                        : `https://placehold.co/1000x1000/png?text=${title}`
+                }
```

Comment:
I don't think we care about this in our example, we can let user handles this, specially because we are "marrying" to this placeholder provider.

Reply:
I added it mostly because nextJS was breaking. But I can conditionally render the image